api/base/parsers.py

```diff
@@ -4,9 +4,11 @@
 from api.base.renderers import JSONAPIRenderer
 from api.base.exceptions import JSONAPIException
 
-
 NO_ATTRIBUTES_ERROR = 'Request must include /data/attributes.'
+NO_RELATIONSHIPS_ERROR = 'Request must include /data/relationships.'
```

Comment:
I'd rather see custom exception classes in `api.base.exceptions` to do this kind of case-specific exception handling, but understand that you are following the existing convention.